File: Navigation/NavigationRailDemo/app/src/main/java/edu/cs4730/navigationraildemo/threeFragment.java
package edu.cs4730.navigationraildemo;


import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import edu.cs4730.navigationraildemo.databinding.FragmentThreeBinding;


/**
 * A simple {@link Fragment} subclass.
 */
public class threeFragment extends Fragment {
    FragmentThreeBinding binding;

    public threeFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
       binding = FragmentThreeBinding.inflate(inflater, container, false);
       return binding.getRoot();
    }

}

File: Advanced/ArchNavDrawerLayout_kt/app/src/main/java/edu/cs4730/archnavdrawerlayout_kt/threeFragment.kt
package edu.cs4730.archnavdrawerlayout_kt

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import edu.cs4730.archnavdrawerlayout_kt.databinding.FragmentThreeBinding

/**
 * A simple [Fragment] subclass.
 */
class threeFragment : Fragment() {
    lateinit var binding: FragmentThreeBinding
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        // Inflate the layout for this fragment
        binding = FragmentThreeBinding.inflate(inflater, container, false)
        return binding.root
    }
}